### PR TITLE
Add redis as session store

### PR DESCRIPTION
**Pull request by bilbof**
This allows users to input over 4kb of data.

https://trello.com/c/CdDdV2fc/120-use-redis-as-session-store

Uses https://github.com/redis-store/redis-rails

### CODE REVIEW

**Comment by theseanything** [comment]
How do we set the Redis URL etc?

File: README.md [hunk 1]
```diff
@@ -35,6 +35,13 @@ Then set up your Docker database
 
 You'll then need to specify the `DATABASE_URL` environment variable before the below tasks.
 
+### Running Redis
+
+#### Locally
+
+    brew install redis
+    redis-server
```
Comment (theseanything) [inline README.md]: Nit: but it think you can use `brew services start redis`

File: Gemfile [hunk 2]
```diff
@@ -4,13 +4,15 @@ ruby File.read(".ruby-version").strip
...
-gem "rails", "~> 6.0.2"
 
 gem "bootsnap", "~> 1"
 gem "dynamoid"
 gem "govuk_app_config", "~> 2.1.1"
 gem "govuk_publishing_components", "~> 21.34.0"
 gem "puma", "~> 4.3"
+gem "rails", "~> 6.0.2"
+gem "redis"
+gem "redis-rails", "~> 5.0"
```
Comment (theseanything) [inline Gemfile]: Do we need this? Seems to be in-built - https://github.com/redis-store/redis-rails#a-quick-note-about-rails-52?

Comment (bilbof) [reply Gemfile]: I wonder if that also applies to sessions? I'm using that gem for the session stuff:

> We are still actively maintaining all other gems in the redis-store family, such as redis-actionpack for session management

**Comment by huwd** [comment]
Humm, why did the deployment pipeline here try and go to heroku?
`coronavirus-redis-sessi-bwz0hn`?

**Review by huwd — APPROVED** [review]
@bilbof might need to bundle install and push?
Conflict in the lockfile.

Otherwise LGTM, if `redis-rails` is still a thing!

File: Gemfile (hunk 2 above)
Comment (theseanything) [reply Gemfile]: Yeah, it's not clear from the documentation. I guess if this works, we can look at it later, if its an issue!

**Review by theseanything — APPROVED** [review]

**Review by rjbaker — APPROVED** [review]
Looks good.

**Pull request merged**